modules/mirror-registry-flags.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-mirroring-installation-images.adoc
+
+[id="mirror-registry-flags_{context}"]
+
+== Mirror registry for {product-title} flags
+
+The following flags are available with local host and remote mirror registries:
+
+[options="header",cols="1,3"]
+|===
+| Flags | Description
+| `--autoApprove` | A boolean value that disables interactive prompts. Will automatically delete quayRoot directory on uninstall. Defaults to false if left unspecified.
+| `--initPassword` | The password of the init user created during Quay installation. Must be at least 8 characters and contain no white space.
+| `--quayHostname` | The value to set `SERVER_HOSTNAME` in the Quay config.yaml. Defaults to `<targetHostname>:8443` if left unspecified. ^[1]^
+| `--quayRoot`, `-r` | The directory where Quay persistent data are saved, including `rootCA.key`, `rootCA,pem`, and `rootCA.srl` certificates. Requires approximately 9 GB for {product-title} Release images, or approximately 668 GB for {product-title} Release images and {product-title} Operator images. Defaults to `/etc/quay-install` if left unspecified.
```

Comment:
Due to different OCP version require different storage size, it is better to mention the ocp version like line 95 of `installing-mirroring-installation-images.adoc`